### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by Yourweatherguy1005YT** [jason0611/FS22_DashboardLive#83] [General]
im worried if im using the right site for dashboard live 
but how do i get the sdk.zip file?

**Comment by Yourweatherguy1005YT** [comment]
and is it ok if i can get some help with setting it up to

**Comment by jason0611** [comment]
Hi, I'm fine with helping you, but I don't understand what's your problem. What do you mean with sdk.zip?

**Comment by cgthunderstrike** [comment]
helllllllp
